packages/schema/lib/schemas/DynamicAuthInputFieldsSchema.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+const AuthInputFieldSchema = require('./AuthInputFieldSchema');
+
+module.exports = makeSchema(
+  {
+    id: '/DynamicAuthInputFieldsSchema',
+    description: 'An array or collection of auth input fields.',
+    type: 'array',
+    items: {
+      oneOf: [{ $ref: AuthInputFieldSchema.id }],
+    },
+    examples: [[{ key: 'abc' }]],
```

Comment:
```suggestion
    examples: [{ key: 'username', isSafe: True}, { key: 'password'}],
```

Now that the FieldSchema is broken, I think we should use specific examples for the purpose it is meant for.

Reply:
That of the DynamicInputFieldsSchema and DynamicOutputFieldsSchema can stay though.